crates/solidity/testing/snapshots/bindings_output/using/binding_enum_member/input.sol

```diff
@@ -0,0 +1,17 @@
+enum Answer {
+  Yes
+}
+
+library Id {
+  function id(Answer ans) returns (Answer) {
+    return ans;
+  }
+}
+
+contract Test {
+  using Id for Answer;
+
+  function testFunc() returns (Answer) {
+    Answer.Yes.id();
```

Comment:
This should resolve for all versions, but you'll need to move the enum inside the `library` because top-level enums are not supported before 0.6.0.